autofit/non_linear/timer.py

```diff
@@ -46,7 +46,7 @@ def update(self):
         """
 
         try:
-            execution_time = str(dt.timedelta(seconds=time.time() - float(self.start_time)))
+            execution_time = str(dt.timedelta(seconds=time.time() - float(self.start_time)).total_seconds())

```

Comment:
Does timedelta need to be used here at all?

Reply:
Apparently not.